app/main.py

```diff
@@ -1,8 +1,42 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
```

Comment:
It's not a good idea to use a class variable to store instances of the class. It's better to handle this outside of the class, in the main part of your program.